modules/ROOT/pages/cloud/overview/overview.adoc

```diff
@@ -0,0 +1,94 @@
+:page-layout: api-partial
+
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. To understand the Cloud API basics, see the xref:ROOT:deploy:deployment-option/cloud/api/cloud-api-overview.adoc[Redpanda Cloud API Overview]. To get started using the API, try the <<quickstart,Quickstart>>.
+
+== Control Plane API
+
+The Control Plane API enables you to programmatically manage your clusters, networks, and resource groups.
+
+The *base URL* of the Control Plane API for all organizations is:
+
+```
+https://api.redpanda.com
+```
+
+== Data Plane APIs
+
+The Data Plane APIs enable you to programmatically manage the resources within your clusters, including topics, users, access control lists (ACLs), and connectors.
+
+The *base URL* of the Data Plane APIs is unique to every cluster. You can retrieve this value by making a xref:get-/v1beta2/clusters/-id-[Get Cluster] request to your target cluster. Use the `dataplane_api.url` from the response body as the base URL when calling the Data Plane API endpoints. 
```

Comment:
Can you link base URL here to https://deploy-preview-283--redpanda-docs-preview.netlify.app/24.1/deploy/deployment-option/cloud/api/cloud-api-overview#data-plane-apis-url?